.github/workflows/pytest.yml

```diff
@@ -28,12 +28,11 @@ jobs:
     runs-on: ${{ matrix.runner }}
     strategy:
       matrix:
-        python-version: ["3.8", "3.11"]
+        python-version: ["3.8", "3.12"]
```

Comment:
Should we also update the version in all actions from 3.11 to 3.12?

Reply:
ah, true